AWSSDKSwiftCLI/Sources/SPR/Process+SPR.swift

```diff
@@ -0,0 +1,36 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Foundation
+import PackageDescription
+import AWSCLIUtils
+
+extension Process {
```

Comment:
These extensions run other utilities to perform steps in publication to the registry:
- describe the contents of a package (used to verify package identity),
- perform archiving of a package version to a .zip file, and
- get the SHA-256 checksum for a file